### DISCUSSION TITLE
Appropriate FOV size for reconstruction?

### DISCUSSION THREAD

**Discussion by imanazinkhah** [villekf/OMEGA#34] [General]
[Hi.]([url](![Screenshot from 2022-07-15 13-47-22](https://user-images.githubusercontent.com/82964793/179193876-5cd58290-433b-40c9-a772-7ace1315bd38.png))) I implemented the geometry of a CT scan using the fan beam method. I did reconstruction with Omega once with 360 projection and a field of view and got a very good image. But after a while, I can't get the previous image by changing the field of view!!! What should be considered to calculate the field of view? Because in reconstruction with different fields of view, very different images are obtained.

**Comment by villekf** [comment]
I'm not sure I understand what is the problem here. How does the image change when you change the FOV? There can be differences, depending on the size of the FOV. If you need to determine the optimal size of the FOV, usually in CT the FOV should be large enough to include all the source-detector combinations/lines (i.e. lines of response).

**Reply by imanazinkhah** [reply]
I have simulated a CT scan with a phantom by GATE. Simulation was done with 360 projection with imageCT.dat. With this information, the reconstruction was done by the OMEGA .
options.xSize = 400/options.binning;
options.ySize = 80/options.binning
options.nProjections = 360
options.dPitch = 1*options.binning;
options.sourceToDetector = 1040;
options.sourceToCRot = 570
options.Nx = 256
options.Ny = 256;
options.Nz = 10;
options.FOVa_x = 350;
options.axial_fov = 340;
Reconstructions were done by choosing different fields of view, but this suitable image of the phantom was obtained only in one field of view. But in the next attempts, I can't get a proper reconstruction. And unfortunately, I don't remember that field of view either. I don't understand what's wrong now. 
![iman](https://user-images.githubusercontent.com/82964793/179261851-0d997503-444f-40a5-abd8-80304ec6dd0c.png)

**Reply by villekf** [reply]
Are you changing the transaxial FOV or axial FOV? Or both? I assume you have tried both decreasing and increasing the size? Since the voxel size is dependent on the FOV size and the number of voxels in the reconstructed image, the current ray tracer might not work properly if the voxel size is too small. Other than that, I really don't know what could be the issue.

**Reply by adiladil23** [reply]
Dear friends, 
 I am trying to use OMEGA but I am not familiar with MATALAB, I successfully run the GATE_CT_MAIN.m but the result was GATE_CT_DATA.mat. 
after reviewing this discussion I see the reconstructed image had the .nii extension. 
Can I have help, please? 
Adil
